etc/scipipe/build_matrix.yaml

```diff
@@ -138,8 +138,8 @@ newinstall:
     git_ref: master
     dir: ''
   docker_registry:
-    repo: lsstsqre/newinstall
-    tag: latest
+    repo: ghcr.io/lsst-sqre/docker-newinstall
+    tag: 9-latest-9.0.0
```

Comment:
So it will be 10-latest-10.0.0 for the next one?

Reply:
Depends whether we go to AlmaLinux 10 or not.

Reply:
Ha. Now I understand that the first 9 and the last 9 aren't the same 9 😄